core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/named/cache/NamedViewCache.java

```diff
@@ -155,9 +151,12 @@ public Iterable<NamedViewDetail> getAllNamedViews(final User user, final String
      *                  name
      * @throws CacheOperationException if the add operation fails
      */
-    public void addToCache(final NamedViewDetail namedView, final boolean overwrite)
-            throws CacheOperationException {
-        super.addToCache(namedView.getName(), namedView, overwrite);
+    public void addToCache(final NamedViewDetail namedView, final boolean overwrite) throws CacheOperationException {
+        try {
+            super.addToCache(namedView.getName(), namedView, overwrite);
+        } catch (final GafferRuntimeException e) {
+            throw new CacheOperationException(e);
+        }
```

Comment:
can we not use unchecked again? If too much of a minefield for this ticket maybe look at exceptions in another ticket

Reply:
Good spot I was meant to back this change out following the other comment about adds to cache should be checked exceptions